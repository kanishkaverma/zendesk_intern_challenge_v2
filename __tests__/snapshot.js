// __tests__/snapshot.js
import React from 'react'
import renderer from 'react-test-renderer'
import Index from '../pages/index'

import * as nextRouter from 'next/router'

nextRouter.useRouter = jest.fn()
nextRouter.useRouter.mockImplementation(() => ({ route: '/' }))

it('renders homepage unchanged', () => {
	const tree = renderer.create(<Index />).toJSON()
	expect(tree).toMatchSnapshot()
})
