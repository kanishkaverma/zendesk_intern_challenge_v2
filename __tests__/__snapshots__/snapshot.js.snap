// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="css-1fri2vb"
>
  <p
    className="chakra-text css-0"
    role="heading"
  >
     Zendesk API is down. kindly retry later. 
  </p>
</div>
`;
